README.Rmd

```diff
@@ -90,8 +90,8 @@ _Note: The ordering numbers below are suggested but don’t all need to strictly
 1. Share company-specific implementations and specifications to be able to harmonize into your design strategy for the package extension. _Here it is important to remain pragmatic and consider a higher perspective than any one company. Engage your company standards representatives and where you find discrepancies across company approaches then question if you really need to be doing things differently here (do health authorities or patients benefit at all if you do?). Also consider that we always expect a level of company-specifics to be covered in the internal company package extensions._
-1. Set up a new public GitHub repo under the [pharmaverse org](https://github.com/pharmaverse) using template https://github.com/pharmaverse/admiraltemplate - this includes set-up pieces (such as CI/CD checks and issue/PR templates) that will enable your package to stay consistent with others in the admiral family, as well as the same core package dependencies and versions. _Note that this step requires org member access which could be granted by of the pharmaverse council reps, who are admins for this org. Also you are free to add additional package dependencies as needed assuming only reliable packages are used, but they must not depend on newer versions of other packages (always reply “no” if updates are suggested during installation)._
-1. Once the repo is available the technical lead could be granted admin access to this repo and then could set up a GitHub team to assign required access for all other co-development team members. Most will only require write access, but you may choose to give the other leads admin access as well so that never a bottle-neck waiting on one person.
+1. Set up a new public GitHub repo under the [pharmaverse org](https://github.com/pharmaverse) using template https://github.com/pharmaverse/admiraltemplate - this includes set-up pieces (such as CI/CD checks and issue/PR templates) that will enable your package to stay consistent with others in the admiral family, as well as the same core package dependencies and versions. See Quick Start Guide for Template section below for instructions. _Note that this step requires org member access which could be granted by of the pharmaverse council reps, who are admins for this org. Also you are free to add additional package dependencies as needed assuming only reliable packages are used, but they must not depend on newer versions of other packages (always reply “no” if updates are suggested during installation)._
+1. Once the repo is available the technical lead could be granted admin access to this repo and then could set up a GitHub team with the same name as the package extension to assign required access for all other co-development team members. Most will only require write access, but you may choose to give the other leads admin access as well so that never a bottle-neck waiting on one person.
```

Comment:
Should we link to what a Github team is? https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team

Just a thought. Looks good to me.

Reply:
No i dont think so - if someone's going to be lead of an admiral package then it's on them to learn the github basics like this and google where they have knowledge gaps etc. These folks should be experienced developers.